docroot/themes/contrib/civictheme/civictheme_library/components/00-base/colors/colors.utils.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Colors component utilities.
+ */
+
+const fs = require('fs');
+const pathUtil = require('path');
+
+const dir = '../../../../dist';
+const basePath = pathUtil.resolve(__dirname, dir);
+const paths = fs.readdirSync(basePath);
+
+function getVariablesCsvPath() {
+  const url = {};
+  paths.forEach((path) => {
+    const csvPath = path.match(/.*\.variables.css/);
+    if (csvPath) {
+      url['path'] = `${dir.replace('../../../', '')}/${path.replace('.css', '.csv')}`;
```

Comment:
Note: 
cannot find civictheme.variables.csv
As its generated at the end. using 
`php civictheme_library/scripts/extract_css_colors_to_csv.php`
Also 
value can be civictheme.variables.csv || styles.variables.csv (sub theme)